### PR TITLE
Create functionality

### PR DESCRIPTION
**Pull request by Jatded1d9l7**

### CODE REVIEW

File: app/controllers/fitness_logs_controller.rb [hunk 1]
```diff
@@ -5,4 +5,17 @@ def index
     render json: fitnesslogs
   end
 
+  def create
+    fitnesslog = FitnessLog.create(log_params)
+    if fitnesslog.valid?
+      render json:fitnesslog
```
Comment (WalkerTexas-coder) [inline app/controllers/fitness_logs_controller.rb]: does "json:" need a space between itself and fitnesslog?

**Review by WalkerTexas-coder — COMMENTED** [review]
small typo comment/ question? also do y'all have a convention for fitnesslog or should it be fitness_log?

File: spec/requests/fitness_logs_spec.rb [hunk 2]
```diff
@@ -15,5 +15,26 @@
...
+          energetic: 7,
+          lethargic: 2,
+          entry: 'crushed it!',
+          user_id: user.id,
+        }
+    }
+      post '/fitness_logs', params: fitness_log_params
+      new_fitness_log = FitnessLog.first
+      expect(response).to have_http_status(200)
+      expect(new_fitness_log.sad).to eq 5
```
Comment (elysemontano) [inline spec/requests/fitness_logs_spec.rb:39]: You should probably test each column in fitness log.

File: app/controllers/fitness_logs_controller.rb (hunk 1 above)
Comment (Jatded1d9l7) [reply app/controllers/fitness_logs_controller.rb]: Added space

File: spec/requests/fitness_logs_spec.rb (hunk 2 above)
Comment (Jatded1d9l7) [reply spec/requests/fitness_logs_spec.rb:39]: Complete

File: spec/requests/fitness_logs_spec.rb [hunk 3]
```diff
@@ -15,5 +15,34 @@
      expect(response).to have_http_status(200)
```
Comment (elysemontano) [inline spec/requests/fitness_logs_spec.rb:15]: Sorry, one more thing...should add tests for the index on each column too.  After that you should be good!

**Review by elysemontano — APPROVED** [review]
Approving, but please make sure to complete the tests for index on each column as well.  The best practice would be to create a separate branch to complete that.  Also, as Austin mentioned, the naming convention for fitnesslog may want to be checked for the future.

**Pull request merged**